target/linux/archs38/config-6.6

```diff
@@ -93,7 +93,6 @@ CONFIG_EXT4_FS=y
 CONFIG_FAT_FS=y
 CONFIG_FB=y
 CONFIG_FB_CORE=y
-CONFIG_FB_IOMEM_FOPS=y
```

Comment:
>> I would suggest not touching these symbols. They should not be manually edited.

> Fixed. I left it everywhere where CONFIG_FB_IOMEM_HELPERS is.

I mean all config files. In kernel 6.6, `CONFIG_FB_IOMEM_FOPS` is selected by `CONFIG_FB_CORE`. These changes will not make any difference.

ref: https://elixir.bootlin.com/linux/v6.6.90/source/drivers/video/fbdev/core/Kconfig#L7

Reply:
Done